huaweicloud/services/evs/resource_huaweicloud_evs_snapshot.go

```diff
@@ -64,11 +71,11 @@ func ResourceEvsSnapshotV2() *schema.Resource {
 	}
 }
 
-func resourceEvsSnapshotV2Create(d *schema.ResourceData, meta interface{}) error {
-	config := meta.(*config.Config)
-	evsClient, err := config.BlockStorageV2Client(GetRegion(d, config))
+func resourceEvsSnapshotV2Create(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	cfg := meta.(*config.Config)
+	evsClient, err := cfg.BlockStorageV2Client(common.GetRegion(d, cfg))
```

Comment:
suggest to use cfg.GetRegion, please check and replace them all

```suggestion
	evsClient, err := cfg.BlockStorageV2Client(cfg.GetRegion(d))
```